source/stylesheets/refills/unstyled/_sliding-panel.scss

```diff
@@ -0,0 +1,36 @@
+.sliding-panel-content {  
+  $sliding-panel-width: 220px;
+
+  @include position(fixed, 0px auto 0px 0px);
+  @include size($sliding-panel-width 100%);
+  @include transform(translateX(- $sliding-panel-width));
+  @include transition(all 0.25s linear);
+  background: #fff;
+  overflow-y: auto;
```

Comment:
Properties should be ordered background, -webkit-overflow-scrolling, overflow-y, z-index